docs/cheatsheet.rst

```diff
@@ -193,7 +193,7 @@ Reserved Keywords
 These keywords are reserved in Solidity. They might become part of the syntax in the future:
 
 ``after``, ``alias``, ``apply``, ``auto``, ``case``, ``copyof``, ``default``,
-``define``, ``final``, ``immutable``, ``implements``, ``in``, ``inline``, ``let``, ``macro``, ``match``,
+``define``, ``final``, ``implements``, ``in``, ``inline``, ``let``, ``macro``, ``match``,
 ``mutable``, ``null``, ``of``, ``partial``, ``promise``, ``reference``, ``relocatable``,
 ``sealed``, ``sizeof``, ``static``, ``supports``, ``switch``, ``typedef``, ``typeof``,
 ``unchecked``.
```

Comment:
Same here.

Reply:
Hey @chriseth not quite sure what you mean. 😓 Can you clarify? I can make another PR to merge.

Reply:
Sorry, this might have required too much inside information: `unchecked` is also a proper keyword by now.

If you want, you can check the file `Token.h` against this list and remove everything from here that is a proper keyword.

Thanks a lot!

Reply:
Gotcha!

Reply:
@chriseth Done in #11698